content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -79,6 +79,13 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 address - see [RFC 4193](https://tools.ietf.org/html/rfc4193))
 {{< /note >}}
 
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+When using an external cloud provider, you can pass a dual-stack `--node-ip` value to
```

Comment:
Better provide an example for the value. Is it a comma-separated list?

Reply:
Ah ... that's fine. So long these details are mentioned/explained somewhere in the page.